src/S3Bucket/Components/DownloadButton.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <VTooltip location="start" :text="downloadError">
+    <template #activator="{ props: activatorProps }">
+      <VIcon
+        v-if="downloadError"
+        v-bind="activatorProps"
+        color="error"
+        size="x-small"
+        icon="mdi-alert-circle"
+        class="mt-1"
+      />
+    </template>
+  </VTooltip>
+  <VBtn
+    icon="mdi-file-download"
+    :disabled="isDisabled"
+    size="x-large"
+    title="Download"
+    @click="downloadFiles"
+  />
+</template>
+
+<script setup>
+import { computed, inject, ref } from 'vue';
+import { convertObjectToFormData } from '../../helpers/axiosHelper';
+import { useBuckets } from '../../helpers/useBuckets';
+/**
+ * @typedef {Object} Props
+ * @property {Array} Props.selectedItems - The selected S3 Bucket items
+ */
+/** @type {Props} */
+const props = defineProps({
+  selectedItems: {
+    type: Array,
+    default: () => []
+  }
+})
+const api = inject('api')
+const { bucketResource, bucketLocation } = useBuckets()
+const downloadError = ref()
+
+// Disabled download if there are no items, or folder items
+const isDisabled = computed(() => {
+  if (
+    props.selectedItems.length === 0 ||
+    props.selectedItems.findIndex((entry) => !entry.is_file) !== -1
+  ) {
+    return true
+  }
+  return false
+})
```

Comment:
Nit: Could be a bit simpler. Something like:
```suggestion
const isDisabled = computed(() => (
  props.selectedItems.length === 0 ||
  props.selectedItems.every((entry) => entry.is_file)
)
```